backend/resources/projects.py

```diff
@@ -118,63 +120,61 @@ def projects_post():
 
     # ToDo: Implement POST /projects
 
-    authToken = request.headers.get('authToken', default=None)
+    auth_token = request.headers.get('authToken', default=None)
     name = request.headers.get('name', default=None)
-    webpage = request.headers.get('webpage', default="")
-    idInstitution = request.headers.get('idInstitution', default="")
+    webpage = request.headers.get('webpage', default=None)
+    id_institution = request.headers.get('idInstitution', default=None)
     goal = request.headers.get('goal', default=None)
-    requiredVotes = request.headers.get('requiredVotes', default=None)
+    required_votes = request.headers.get('requiredVotes', default=None)
     until = request.headers.get('until', default=None)
-    milestones = request.headers.get('milestones', default="")
+    milestones = request.headers.get('milestones', default="[]")
 
-    if authToken is None:
+    if auth_token is None:
         return jsonify({'error': 'Not logged in'}), 403
-    
-    if name is None or goal is None or requiredVotes is None or until is None:
-        return jsonify({'error': 'Missing parameter'}), 403
 
-    session = DB_SESSION()    
+    if name is None or goal is None or required_votes is None or until is None:
+        return jsonify({'error': 'Missing parameter'}), 403
 
-    # TODO uuid refactor + smartcontract_id
+    session = DB_SESSION()
 
     max_ms_v = int(session.query(func.max(Milestone.idMilestone)).one()[0])
     max_pr_v = int(session.query(func.max(Project.idProject)).one()[0])
 
     try:
         session.add(Project(
-            idProject = str(max_pr_v + 1),
-            nameProject = name,
-            webpageProject = webpage,
-            smartcontract_id = 1,
-            institution_id = idInstitution
-            ))
+            idProject=(max_pr_v + 1),
+            nameProject=name,
+            webpageProject=webpage,
+            smartcontract_id=1,  # ToDo: ist das immer 1?
```

Comment:
Nein, wir müssen aber auf die Integration der SmartContracts warten. Wir speichern immer einen Verweis auf den SmartContract und dieser muss dann dementsprechend beim anlegen mit eingetragen werden.

Reply:
ah, got it